src/base/bittorrent/peerinfo.cpp

```diff
@@ -273,131 +273,136 @@ qreal PeerInfo::relevance() const
 
 void PeerInfo::determineFlags()
 {
+    const QString nonBreakingSpacesEquality = QString::fromUtf8(C_NON_BREAKING_SPACE)
```

Comment:
why `C_NON_BREAKING_SPACE` is required? will it break to multiple lines without it?
If not then the regular ascii space is sufficient.

Reply:
I don't really know, just to be sure. Of course i'll remove it if you think it's unnecessary.

Reply:
How can you not really sure what you wrote?
Either you are really confident it is needed or just don't bother touch it.

Reply:
Also fromUtf8 is redundant in Qt5.

Reply:
>nonBreakingSpacesEquality

Why this confusing name? Why not just nonBreakingSpace?
Also I don't see `:` at end of this line.

Reply:
>Either you are really confident it is needed or just don't bother touch it.

Duly noted. Excuse me for trying to not do half a job.
>Why this confusing name? Why not just nonBreakingSpace?

Because it's not just a non-breaking space?
>Also I don't see : at end of this line.

Why should there be?
>Also fromUtf8 is redundant in Qt5.

Then what? `QString(C_NON_BREAKING_SPACE)`?

Reply:
>Why should there be?

Oops... I see only unterminated statement and several blank lines in my browser because of this monstrous indentation.

Reply:
And I mean you forget to add ;

Reply:
>Then what? `QString(C_NON_BREAKING_SPACE)`?

`QString(C_NON_BREAKING_SPACE)` in cases when explicit QString constructor is required, or `C_NON_BREAKING_SPACE` when it isn't.